### PR TITLE
[CLOUDGA-8952] Remove memory for nodeconfig

### PR DESCRIPTION
**Pull request by posriniv**

### CODE REVIEW

File: managed/resource_cluster.go [hunk 1]
```diff
@@ -308,6 +303,7 @@ func createClusterSpec(ctx context.Context, plan Cluster, clusterExists bool) (c
 		if clusterType == "SYNCHRONOUS" {
 			info.PlacementInfo.SetMultiZone(false)
 		}
+		info.SetIsDefault(false)
```
Comment (cdavid) [inline managed/resource_cluster.go:306]: What is this for?

File: managed/common.go [hunk 2]
```diff
@@ -25,8 +25,28 @@ func getProjectId(accountId string, apiClient *openapiclient.APIClient) (project
...
+		return 0, false, string(b)
+	}
+	instanceData := instanceResp.GetData()
+	nodeConfigList, ok := instanceData[region]
+	if !ok || len(nodeConfigList) == 0 {
+		return 0, false, "No nodes configured for the given region."
+	}
+	for _, nodeConfig := range nodeConfigList {
+		if nodeConfig.GetNumCores() == numCores {
+			memory = nodeConfig.GetMemoryMb()
```
Comment (cdavid) [inline managed/common.go:41]: I would add logs around this area (in particular before the `return` statements).

**Review by cdavid — COMMENTED** [review]
lgtm otherwise.

File: managed/resource_cluster.go (hunk 1 above)
Comment (posriniv) [reply managed/resource_cluster.go:306]: It shows the error `IsDefault` has to be fault for multi-region clusters, I am not exactly sure of the reason.

File: managed/common.go (hunk 2 above)
Comment (posriniv) [reply managed/common.go:43]: Added the logs.

**Review by cdavid — APPROVED** [review]

File: managed/resource_cluster.go (hunk 1 above)
Comment (posriniv) [reply managed/resource_cluster.go:306]: The default value was changed from `true` to `false` here https://github.com/yugabyte/yugabyte-cloud/commit/3c49c65e19d36b444f745b2b8b6035348cdc0b46#diff-4ea2fce92e955dee25f775c4ad6467aac4d54cf4ca03dfdf09a27509ed58082cL3654-R3657. That is why, we will have to make this change.

**Pull request merged**